### PR TITLE
translation refactoring of en-us.xaml for YostarEN

### PR DESCRIPTION
**Pull request by Constrat**
I've checked the entire en-us.xaml and tried to give it uniformity while correcting some grammatical and logical mistakes.

### CODE REVIEW

**Comment by MistEO** [comment]
LGTM. @ABA2396 please help to check.

File: src/MaaWpfGui/Res/Localizations/en-us.xaml [hunk 1]
```diff
@@ -313,32 +313,32 @@ At the end of the level, enter "Normal/Hard" to switch between Normal and Tough
...
     <system:String x:Key="LSTip">LS (Battle Record)</system:String>
     <system:String x:Key="SKTip">SK (Carbon)</system:String>
     <system:String x:Key="PR-ATip">PR-A (Med/Def Chip)</system:String>
     <system:String x:Key="PR-BTip">PR-B (Cst/Sni Chip)</system:String>
     <system:String x:Key="PR-CTip">PR-C (Pio/Sup Chip)</system:String>
     <system:String x:Key="PR-DTip">PR-D (Grd/Spc Chip)</system:String>
-    <system:String x:Key="Pormpt1">It's time to complete Annihilation</system:String>
-    <system:String x:Key="Pormpt2">Don't forget Annihilation</system:String>
+    <system:String x:Key="Pormpt1">It's Monday: time to clear the Annihilation</system:String>
+    <system:String x:Key="Pormpt2">It's Sunday: don't forget about Annihilation</system:String>
```
Comment (ABA2396) [inline src/MaaWpfGui/Res/Localizations/en-us.xaml:327]: This looks a bit long, have you tested if it displays properly? I'm not sure if the current control supports automatic line feeds

**Review by ABA2396 — APPROVED** [review]

File: src/MaaWpfGui/Res/Localizations/en-us.xaml (hunk 1 above)
Comment (ABA2396) [reply src/MaaWpfGui/Res/Localizations/en-us.xaml:327]: I tested and found that had added auto line feeds before, so that's fine

Comment (Constrat) [reply src/MaaWpfGui/Res/Localizations/en-us.xaml:327]: Perfect!

**Pull request merged**